app/src/main/res/layout/fragment_feed.xml

```diff
@@ -81,4 +82,51 @@
         </android.support.constraint.ConstraintLayout>
     </android.support.v7.widget.CardView>
 
+    <FrameLayout
+        android:id="@+id/card_donation"
+        android:layout_width="0dp"
+        android:layout_height="102dp"
+        android:layout_marginBottom="90dp"
+        android:layout_marginEnd="39dp"
```

Comment:
Try not to use odd numbers. cz all screen have even numbers of pixels.

Reply:
like in design